x-pack/plugins/watcher/public/plugin.ts

```diff
@@ -85,9 +83,16 @@ export class WatcherUIPlugin implements Plugin<void, void, Dependencies, any> {
     home.featureCatalogue.register(watcherHome);
 
     licensing.license$.pipe(first(), map(licenseToLicenseStatus)).subscribe(({ valid }) => {
+      // NOTE: We enable the plugin by default instead of disabling it by default because this
+      // creates a race condition that can cause the app nav item to not render in the side nav.
+      // The race condition still exists, but it will result in the item rendering when it shouldn't
+      // (e.g. on a license it's not available for), instead of *not* rendering when it *should*,
+      // which is a less frustrating UX.
       if (valid) {
         watcherESApp.enable();
```

Comment:
Can we remove this line now too?

Reply:
Whoops sorry, I hit merge before I saw this comment! You're right, that line isn't necessary. Let's remove it in a subsequent PR, or perhaps leave it if we refactor this code to consume all events in the stream and not just the first one.